kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/Exceptions.kt

```diff
@@ -88,3 +88,12 @@ class ParseException(var reason: String) : ExtensionsException() {
 
     override fun toString(): String = reason
 }
+
+/**
+ * Throws when invalid time unit given to duration parser.
+ *
+ * @param unit Invalid unit.
+ */
+class InvalidTimeUnitException(var unit: String) : ExtensionsException() {
```

Comment:
Like I said, that would be better if we used a common base exception for all the parsers, maybe even define them in the `parsers` folder